var/spack/repos/builtin/packages/hydrogen/package.py

```diff
@@ -90,6 +90,7 @@ class Hydrogen(CachedCMakePackage, CudaPackage, ROCmPackage):
     # Note that this forces us to use OpenBLAS until #1712 is fixed
     depends_on("openblas", when="blas=openblas")
     depends_on("openblas +ilp64", when="blas=openblas +int64_blas")
+    depends_on("openblas@0.3.21:0.3.23", when="blas=openblas arch=ppc64le:")
```

Comment:
(not a request, just a question) The `blas` variant is a bit weird. Is there any reason why this is / was necessary?

Reply:
@alalazo We find that many versions of `openblas` fail to build on power architectures.  This is to help preserve sanity.

Reply:
@bvanessen Fine, this was just a curiosity - but I was asking about the pre-existing `blas=XXX` variant, not about the version range :thinking: